tests/test_cc_socket_can.py

```diff
@@ -191,54 +191,76 @@ def test_cc_close(logging_requested, mock_can_bus, mocker):
 
 
 @pytest.mark.parametrize(
-    "parameters",
+    "message,remote_id",
     [
-        {"msg": b"\x10\x36", "remote_id": 0x0A, "raw": True},
-        {"msg": b"\x10\x36", "remote_id": None, "raw": True},
-        {"msg": b"\x10\x36", "remote_id": 10, "raw": True},
-        {"msg": b"", "remote_id": 10, "raw": True},
-        {"msg": message_with_tlv, "remote_id": 0x0A, "raw": False},
-        {"msg": message_with_no_tlv, "remote_id": 0x0A, "raw": False},
-        {"msg": message_with_no_tlv},
-        {"msg": message_with_no_tlv, "remote_id": 36, "raw": False},
+        (
+            b"\x10\x36",
+            0x0A,
+        ),
+        (
+            b"\x10\x36",
+            None,
+        ),
+        (
+            b"\x10\x36",
+            10,
+        ),
+        (
+            b"",
+            10,
+        ),
+        (
+            message_with_tlv,
+            0x0A,
+        ),
+        (
+            message_with_no_tlv,
+            0x0A,
+        ),
+        (message_with_no_tlv, None),
+        (message_with_no_tlv, 36),
     ],
 )
-def test_cc_send(mock_can_bus, parameters):
-
+def test_cc_send(mock_can_bus, message, remote_id):
+    if isinstance(message, Message):
+        message = message.serialize()
     with CCSocketCan(remote_id=0x0A) as can:
-        can._cc_send(**parameters)
+        can._cc_send(message, remote_id=remote_id)
```

Comment:
How is that possible... Its a dict in the background...

Reply:
I don't the problem of calling the _cc_send with message and remote id

Reply:
`_cc_send`'s msg parameter has never been a dict, it's only the CCProxy that takes any args/kwargs